wooey/templates/wooey/base.html

```diff
@@ -538,10 +551,23 @@
                                 $field = $field2;
                             }
                         }
-                        if ($form.hasClass('form-inline') || $form.hasClass('navbar-form'))
+                        if ($form.hasClass('form-inline') || $form.hasClass('navbar-form')) {
                             addInlineError($field, error);
-                        else
+                        } else
                             addUlError($field, error);
+
+                        // See if we're in a tabbed view; find the parent tab and set it to error state
+                        $tab_pane = $field.closest('.tab-pane')
```

Comment:
I like the tab pane modification a lot. You left out a semi-colon here btw as well.